src/settings.js

```diff
@@ -1682,6 +1682,21 @@ var IMPORTED_MEMORY = 0;
 // to loading split modules.
 var SPLIT_MODULE = 0;
 
+// How to calculate reverse dependencies (dependencies from JS functions to
+// native functions) prior to linking native code with wasm-ld.  This option
+// has three possible values:
+// 'auto': (default) inspect the object code passed to the linker (by running
+//         llvm-nm) all all inputs and use the map in deps_info.py to
+//         determines the set of additional dependencies.
```

Comment:
```suggestion
//         llvm-nm) on all inputs and include exactly the ones that
//         are needed (by using the map in deps_info.py to
//         determines the set of additional dependencies).
```

Reply:
(focuses on the user-noticeable side, leaving the technical details to parentheses)